### ISSUE TITLE
Dead account... Can I have it?!

### ISSUE THREAD

**Issue by jordanpoulton** [HoneyBadger/gettingandcleaningdata#1]
I noticed this account hasn't been used in quite a few years... Any chance you'd be willing to let me take it over? I have a project I'm working on and this would be the ideal name for our account.

Let me know, thanks :) 

J

**Comment by jordanpoulton** [comment]
PS. Feel free to contact me at jordan@pwnylabs.com

**Comment by HoneyBadger** [comment]
Yeah man. You can have it. I could never figure out this GitHub shit
anyways. What do I need to do?

**Comment by jordanpoulton** [comment]
Sorry for slow reply! Thanks so much, this account will be useful to a project I'm working on. 

I think the best practise would be to update the email associated to the account to mine:

jim@pwnylabs.com

and then change the password to something simple ( temppassword12345 ), email me to confirm the password, then I'll log in and update all the details to something new.

How does that sound?

Thanks again,
J
